webboot/webboot.go

```diff
@@ -148,15 +132,25 @@ func main() {
 
 	//Processes the URL to receive an io.ReadCloser, which holds the content of the downloaded file
 	log.Println("Retrieving the file...")
-	read, err := linkOpen(URL)
+	iso, err := linkOpen(URL)
 	if err != nil {
 		log.Fatal(err)
 	}
 
-	err = write(read, filename)
+	// TODO: Find a persistent memory device large enough to store the content. If no blocks are available, error the user.
+	pmem, err := os.OpenFile("/dev/pmem0", os.O_APPEND|os.O_WRONLY, 0600)
 	if err != nil {
 		log.Fatal(err)
 	}
+	if _, err := io.Copy(pmem, iso); err != nil {
+		log.Fatalf("Error copying to persistent memory device: %v", err)
+	}
+	if err = iso.Close(); err != nil {
+		log.Fatalf("Error closing reader for %v: %v", URL, err)
```

Comment:
I like this style of error message because this will let the user know that the reader for a URL was having trouble closing and listing the specific URL will let the user know which specific URL was having problem closing. However, I am open to suggestions for changing the printing style :)